dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterServerLoadProtection.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.server.master.config;
+
+import org.apache.dolphinscheduler.meter.metrics.SystemMetrics;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class MasterServerLoadProtection {
+
+    private boolean enabled = true;
+
+    private double maxCpuUsagePercentageThresholds = 0.7;
+
+    private double maxJVMMemoryUsagePercentageThresholds = 0.7;
+
+    private double maxSystemMemoryUsagePercentageThresholds = 0.7;
+
+    private double maxDiskUsagePercentageThresholds = 0.7;
```

Comment:
keep the same default value with config.
```suggestion
    private double maxCpuUsagePercentageThresholds = 0.8;

    private double maxJVMMemoryUsagePercentageThresholds = 0.8;

    private double maxSystemMemoryUsagePercentageThresholds = 0.8;

    private double maxDiskUsagePercentageThresholds = 0.8;
```

Reply:
Good catch, done.